docs/preproduction/vision-document.md

```diff
@@ -0,0 +1,70 @@
+
+## Document Structure
+
+The style and structure you choose to outline your document with matters less than the actual information, because format tends to vary from one developer/publisher to the next. Below are some examples of vision documents and how you may want to organize them:
+
+[Example 1]("../assets/vision-docs/on_design_documents.pdf") | [Example 2]() | [Example 3]()
+
+## The Information to Outline
+
+To start, I'm going to walk through some of the information you might want to include in this document that is most important to know about your game. 
+
+### The Key Info 
+
+At the very top of the vision document, you will want to include some bullet points or a short section with the most relevent information. 
+
+* **Game Name**
+* **Genre** - 3D Action/FPS? 2D Puzzle platformer?
+* **Platform** - iOS, PC, WebGL, etc
+* **Target Audience** - age range or specific demographic?
+* **Engine** - we are using Unity Engine
+* **Controls/Input** - WASD, touch screen, etc
+* **Number of Players** - single player, 4-person coop, etc
+
+Games of particular platforms or genres may require additional specifics. For instance, you would want to specify an aspect ratio and device orientation for a mobile game. 
+
+### High Level Concept 
+
+This is a basic overview description of the core gameplay elements and key info.
+
+Here's an example quote from Diablo I:
+
+> *Diablo is a role playing game wherein a player creates a single character and guides him through a dungeon in an attempt to find and destroy 'Diablo', the devil himself. All the action takes place in an isometric, three-quarter perspective, with diamond-shaped, 'square' floor spaces (see diagram).*
+
+[Full Document]("http://www.graybeardgames.com/download/diablo_pitch.pdf")
+
+### Gamplay Overview
+
+This section should describe how to play your game and what the player will be doing to progress. The reader should be able to fully understand the gameplay 
+
+* Controls
+* Core game-loop
+* Main mechanic(s)
+* Progression
+* Win/lose conditions
+
+### Walkthrough
+
+ Imagine a walkthrough you'd read online when you get stuck in a game. It tells you the critical path, explains the moment to moment gameplay. Maybe even has some light maps or other visual aids. This forces the designer to really visualize how the game works in concrete terms. Do a walkthrough for a potential puzzle or typical section of gameplay.
+
+ Examples:
+ [Example 1]("../assets/vision-docs/Vision_doc.pdf")
+
+### Additional Sections
+* Setting and Narrative
+* In-Game Economy
+* Art & Sound Direction
+* Budget Breakdown
```

Comment:
I don't think budget is worth including. Typical devs ("below the line" workers to borrow a film term https://en.wikipedia.org/wiki/Below-the-line_(filmmaking) ) will never know this info and it's really irrelevant to most unfunded student projects